### ISSUE TITLE
Feedly support?

### ISSUE THREAD

**Issue by kutsan** [akrennmair/newsbeuter#518]
I know, newsbeuter not intended to be Feedly client but still would be nice to have Feedly integration.

**Comment by Minoru** [comment]
Yeah, it's been actively requested back in #63. I took a look at Feedly API some time ago—they use OAuth, for which we don't have support in Newsbeuter yet, so adding Feedly isn't quite as simple as adding some other APIs was.

I'm not working on this at the moment, but if you—or anyone else—want to, I'm glad to discuss and help with refactoring Newsbeuter to add Feedly to the mix.

**Comment by lony** [comment]
+1

**Comment by Minoru** [comment]
@lony, you just mailed 55 people. Please use [reactions](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments) instead.
